cmd/arena/commands/trainer_mpi.go

```diff
@@ -524,6 +539,48 @@ func (tt *MPIJobTrainer) isMPIPod(name, ns string, item v1.Pod) bool {
 	return true
 }
 
+func (tt *MPIJobTrainer) resources(name string, namespace string, pods []v1.Pod) ([]Resource, error) {
```

Comment:
It will only be used when init the `MPIJobTrainer` . 
If other func want to get a job's resources, it can call `Resources` public function to get the resources member of `BasicJobInfo` struct.